extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaTuningConfig.java

```diff
@@ -57,7 +62,8 @@ public KafkaTuningConfig(
       @JsonProperty("buildV9Directly") Boolean buildV9Directly,
       @JsonProperty("reportParseExceptions") Boolean reportParseExceptions,
       @JsonProperty("handoffConditionTimeout") Long handoffConditionTimeout,
-      @JsonProperty("resetOffsetAutomatically") Boolean resetOffsetAutomatically
+      @JsonProperty("resetOffsetAutomatically") Boolean resetOffsetAutomatically,
+      @JsonProperty("outputMediumFactory") @Nullable OutputMediumFactory outputMediumFactory
```

Comment:
IMO either we add `Nullable` to all fields or no reason to have it on `OutputMediumFactory`.

Reply:
Added `@Nullable` to all params